test/required-headings-wrong-match-case.md

```diff
@@ -0,0 +1,9 @@
+# Title
+
+## First heading {MD043}
+
+## Second Heading
+
+### Random heading
+
+## Third Heading
```

Comment:
And all upper for this (adding "{MD043}"?

Reply:
I pushed a commit for this - I'd forgotten only the first wrong heading gets flagged.